From 041ccc2af2406884f3b1f5efc6bdb51952db5c8c Mon Sep 17 00:00:00 2001
From: Aron Budea <aron.budea@collabora.com>
Date: Mon, 21 Apr 2025 17:23:26 +0930
Subject: tdf#165917: Revert "tdf#165228 Don't reuse previous path in save
 dialog"

This has been partially resolved in 25.8 by
3fa39a4dadc8e2777185465a6f7c9968c8cf44d1.

Since that is only a partial fix (see eg. tdf#165392), and was
not backported, let's revert in release versions due to number
of reports.

This reverts commit 67cffdf1b2872b4bcac657f7bcf63f19c261a5c6.

Change-Id: I69f230ef1fca77d5636feec0895b9ca81346f4a1
Reviewed-on: https://gerrit.libreoffice.org/c/core/+/184405
Tested-by: Jenkins
Tested-by: Gabor Kelemen <gabor.kelemen.extern@allotropia.de>
Reviewed-by: Aron Budea <aron.budea@collabora.com>
(cherry picked from commit 446fe1d1b85ee98a4b8146c921e7eea22ed7e3a5)
Reviewed-on: https://gerrit.libreoffice.org/c/core/+/184503
Reviewed-by: Christian Lohmaier <lohmaier+LibreOffice@googlemail.com>
Tested-by: Christian Lohmaier <lohmaier+LibreOffice@googlemail.com>
Reviewed-by: Xisco Fauli <xiscofauli@libreoffice.org>
---
 sfx2/source/dialog/filedlghelper.cxx | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sfx2/source/dialog/filedlghelper.cxx b/sfx2/source/dialog/filedlghelper.cxx
index dce3103de3cc..07325adf3085 100644
--- a/sfx2/source/dialog/filedlghelper.cxx
+++ b/sfx2/source/dialog/filedlghelper.cxx
@@ -1476,7 +1476,6 @@ ErrCode FileDialogHelper_Impl::execute( std::vector<OUString>& rpURLList,
     }
 
     rpURLList.clear();
-    maPath.clear(); // tdf#147202 This should not survive between calls to execute
 
     if ( ! mxFileDlg.is() )
         return ERRCODE_ABORT;
-- 
cgit v1.2.3